### ISSUE TITLE
Confused on how to load in Python

### ISSUE THREAD

**Issue by abrad1212** [openimages/dataset#41]
I have been trying for a few hours to load the data into python for training with Keras. Is there a tutorial or can someone please guide me on how I can go about doing this.

Thanks,
-Alex

**Comment by rkrasin** [comment]
One working example is https://github.com/openimages/dataset/blob/master/tools/classify.py with the following doc: https://github.com/openimages/dataset/wiki/Running-a-pretrained-classifier

As for training, you will most likely hit #40.

**Comment by abrad1212** [comment]
@rkrasin Thanks, so in #40 It seems like there is no way to train this? But if that is not right then would work 

`X = np.array(imgs, dtype='float32')`
`Y = np.eye(NUM_CLASSES, dtype='uint8')[labels]`

**Comment by abrad1212** [comment]
@rkrasin Ohhh and why would you ever wanna leave Google, as you mentioned in #40

**Comment by rkrasin** [comment]
>@rkrasin Thanks, so in #40 It seems like there is no way to train this?

Correct. At least, I don't know of a way. It can be fixed, but I think it's worth waiting for the models trained on v2, as hinted in [README.md](https://github.com/openimages/dataset/blob/master/README.md#announcements): ```Coming soon: Trained models (both image-level and object detectors)```.

The models are likely be more accurate, as the V2 data is much cleaner.

As for the release date, I have no information about that. My guess would be 2-3 more weeks before we see anything.

>But if that is not right then would something like this work
>
>X = np.array(imgs, dtype='float32')
>Y = np.eye(NUM_CLASSES, dtype='uint8')[labels]

I am sorry, I didn't get the question here.

>Ohhh and why would you ever wanna leave Google

https://www.linkedin.com/in/ivan-krasin-0ba73ba/

**Comment by abrad1212** [comment]
@rkrasin 
> I am sorry, I didn't get the question here.

I meant if I wasn't right about not being able to train with the dataset, then would something like that code work.

But I have one last question, why the hell would Google make a dataset public that we can not use to train?

**Comment by rkrasin** [comment]
>But I have one last question, why the hell would Google make a dataset public that we can not use to train?

The dataset itself is fine. You can use it for training, with any framework you want.

The dataset comes with a pretrained classifier and a working example (in Python + Tensorflow).
The pretrained classifier seems to have a bug that prevents using it as is for finetuning. It's an issue, but not a critical one: there are myriads of pretrained classifiers on the web, most of which can be used for finetuning. The value of the dataset is in the ability to train your own classifier from scratch. And there's nothing that prevents from doing this.
